src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -16556,7 +16556,6 @@ namespace Google.Example.Library.V1
         /// </summary>
         /// <param name="resource">
         /// REQUIRED: The resource which the label is being added to.
-        /// In the form "shelves/{shelf_id}/books/{book_id}".
```

Comment:
Why are these deleted all over the place? (i.e. please explain the connection between the code change and this change in the generated comments)

Reply:
I changed the pattern of the field `resource` from `Book` to `*` to repro this issue: https://github.com/googleapis/gapic-generator/pull/3138/files#diff-4c8eaf76addf344f8abf6b2800029f9eR51

And the comment is no longer correct... so I deleted the comment, hence the change is baseline.